### ISSUE TITLE
The volume is set to maximum after pausing Kodi for a few minutes

### ISSUE THREAD

**Issue by mYnDstrEAm** [xbmc/xbmc#18602]
### Describe the bug
When resuming a video after having pressed pause it has the volume set to the maximum.

This does not occur when pausing for just a short while - it only appears when pausing for a few minutes until the screen gets dimmed a bit (I haven't tested if it occurs before the screen gets dimmed yet).

It might be related to the audio setting to keep the audio device active. If changing this option solves this issue the volume should not be set to the max (with any other setting than "always") after pausing nevertheless.

## Expected Behavior
The volume should be exactly as before (when one pressed pause).

## Actual Behavior
The volume is set to the maximum.

### To Reproduce

1. Play a video (filename.mp4 in the logs)
2. Pause the video (spacebar)
3. Wait a short while (like until the screen gets dimmed)

### Debuglog

The debuglog can be found here:
https://pastebin.com/GkDuXDag

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [x] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Debian 10 stable
 - Kodi version: 17.6

**Comment by fuzzard** [comment]
You'll need to test and provide logs from a Kodi v19 (Alpha 2 or a nightly). Nothing will be done for 17.6, and im also told debian go a bit off script with their packages, but i'll let other linux users/devs chime in on that.

**Comment by mYnDstrEAm** [comment]
I really would like to use v19 but 17.6 is the latest version that I could get under Debian. I asked about how to get a newer or the latest version running under Debian [here](https://unix.stackexchange.com/questions/584798/how-to-install-latest-kodi-on-latest-debian). The only way seems to be [via flatpak](https://www.flathub.org/apps/details/tv.kodi.Kodi) and I don't want to do this (or have people required to do this) mostly for security reasons that include the issues described [here](https://github.com/flathub/flathub/issues/1498). If you know of any other way please add a reply to the question. I'm interested what other users/devs have to say about it and maybe I'll use flatpak at some point even though I think the version in Debian's repos should work fine and if it isn't the repos most likely need a newer version.

**Comment by DaVukovic** [comment]
As also said at the other issue (and I will close this one as well), you are able to compile Kodi from source. Start reading here: 

https://github.com/xbmc/xbmc/blob/master/docs/README.Linux.md

> think the version in Debian's repos should work fine and if it isn't the repos most likely need a newer version.

Nothing to say against your expectations, but we have to admit that those are not correct ;)
There's a reason we don't support the "Kodi from Debian" version as Debian changes our code to their likings. Hence we can't help much in that regard and if there's an issue with the Kodi version you have installed from Debian, then please feel free to open an issue at their bugtracker. 

Won't comment on the flatpack issue/problems. 

Please open a bugreport with a supported Kodi version either compiled from source, or in use with Ubuntu where you are able to install Kodi from our PPA. If you don't want to use Ubuntu (which is somewhat understandable), then you have no other choice than to compile Kodi from source like many other users are doing it.

**Comment by howie-f** [comment]
all untested, you can try to install the "Kodi from Debian" sid - package to buster, which is 18.8. should also be possible to build the debian source package and leave you with a functional build environment.
still these versions are not supported because debian uses its system-ffmpeg and does other patches.

if you want a vanilla matrix on buster, then there's no other way than building from source like @DaVukovic mentioned

**Comment by mYnDstrEAm** [comment]
I have built v19 alpha 2 now. I created a separate issue for the problems with the guide that I had: #18612
However, I think things should work fine with the version in Debian's repositories.
One thing I noticed with the latest version is that there's no reddit add-on - why is that?

And I still have this issue with the latest version - should I create a new issue for it or should this be reopened and edited?
